crates/std/src/line_hash.rs

```diff
@@ -2,23 +2,10 @@
 #[macro_export]
 #[doc(hidden)]
 macro_rules! line_hash {
-    ($s:expr) => {{
-        use std::collections::hash_map::DefaultHasher;
-        use std::hash::{Hash, Hasher};
-
-        let id = $s;
-
-        let mut s = DefaultHasher::new();
-        id.hash(&mut s);
-        s.finish()
+    (($s:expr),*) => {{
+        format!("{:?}", $($s),*)
     }};
     () => {{
-        let id = concat!(file!(), line!(), column!());
-        line_hash!(id)
-    }};
-    ($($s:expr),*) => {{
-        let mut s: u128 = 0;
-        $(s += $crate::line_hash!($s) as u128;)*
-        $crate::line_hash!(s)
-    }};
+        concat!(file!(), line!(), column!())
```

Comment:
Hm, this isn't a hash; I don't know if that actually matters but maybe we should rename it to `line_uid` or something

Reply:
Good point. Realized it would be far more descriptive to find out the origin of a gizmo scope rather than prehash it